src/ontology/cl-edit.owl

```diff
@@ -28247,6 +28248,17 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16230044"^^xsd:string) r
 AnnotationAssertion(rdfs:label obo:CL_4030025 "renal cortical fibroblast")
 EquivalentClasses(obo:CL_4030025 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005270)))
 
+# Class: obo:CL_4032000 (club-like cell of the urethra epithelium)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30566875"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:31390564"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:32497356"^^xsd:string) obo:IAO_0000115 obo:CL_4032000 "An epithelial cell of the urethra that is similar to lung club cells in their expression of SCGB1A1. Club-like cells of the urethra epithelium are similar to lung club cells in their enrichment of immunomodulatory programs.")
```

Comment:
I only skimmed the references, but did not find mention of this cell in PMID:31390564. Did I miss it?

Reply:
Free text definition for your consideration:
"An epithelial cell of the urethra that has an expression profile similar to lung club cells. (+ gloss)"

Reply:
not gonna lie, i just took the requestors word for it lol, but thanks for checking, will double check and remove if it is not in there. I like your suggestion for def, gives it room if things changes, will change accordingly, thanks.